Security/src/CVE-2023-23397/CVE-2023-23397.ps1

```diff
@@ -459,13 +468,22 @@ begin {
         )
 
         try {
-            if ($null -ne (Get-InstalledModule -Name "AzureAD" -ErrorAction SilentlyContinue)) {
-                Import-Module "AzureAD" -ErrorAction Stop
-                Write-Host "`nPrompting user for authentication, please minimize this window if you do not see an authorization prompt as it may be in the background"
-                Connect-AzureAD -AzureEnvironmentName $AzureEnvironmentName -ErrorAction Stop
-            } else {
-                throw "AzureAD module was not found on this computer. You can install it by running Install-Module AzureAD"
-            }
+            if ($null -ne (Get-InstalledModule -Name "AzureAD" -ErrorAction SilentlyContinue))
+			{
+				Import-Module "AzureAD" -ErrorAction Stop
+				Write-Host "`nPrompting user for authentication, please minimize this window if you do not see an authorization prompt as it may be in the background"
+				Connect-AzureAD -AzureEnvironmentName $AzureEnvironmentName -ErrorAction Stop
+			}
+			elseif ($null -ne (Get-InstalledModule -Name "AzureADPreview" -ErrorAction SilentlyContinue))
```

Comment:
Same here, please revert this change and remove the `AzureADPreview`

Reply:
Same as above I have tested the AzureADPreview module and it works correctly. It is just an elif statement, why would you not manually test it by installing the AzureADPreview version?